### ISSUE TITLE
combine two nav's into one?

### ISSUE THREAD

**Issue by mattmagi** [lukaszfiszer/selectnav.js#8]
I have two main nav's and I have set both to have the same id "nav" but when the drop down is active it only takes the first nav...is there a way I can combine both into ONE dropdown?

Thanks in advance!

**Comment by lukaszfiszer** [comment]
Hi mattmagi,

Sorry for late reponse, but I've just restarted work on this project after a few months of break.

This will not work because basically you should never have more than one element with the same id on a single page. I might consider adding support for a class selector, but this will give you two seperate dropdown menus and not one combined. Anyway, combining many `<ul>` into one dropdown was not a use case I imagined. Could you provide some more details on what and why you want to achieve this? Come screenshot or code posted on jsfiddle?

thanks

**Comment by rusticguy** [comment]
Hi,

Pl refer to this:
https://github.com/brewern/mobile-menu

I have tested this on a site and it also works great combining multiple menus into one menu on mobile interface.
This could be useful for cases where on desktop the entire navigation is broken down into 2 or more lists which need to be combined on mobile so that one has everything at one place.

**Comment by lukaszfiszer** [comment]
Yeah, that make sense. A good use case for me would be a navigation and language list combined into one nav for mobile. I added it to my roadmap, stay tuned.

**Comment by beside4ever** [comment]
Hi mattmagi,
I suggest a hack to do this: (which works for me)
Set a hidden `<ul>` in your header and place any `<li>`s you want in it.
